common/playAbility.cpp

```diff
@@ -342,7 +342,7 @@ Resumable ServerPlayer::checkTiming() {
 
         if (playableCount > 1) {
             clearExpectedComands();
-            addExpectedCommand(CommandPlayAbility::GetDescriptor()->name());
+            addExpectedCommand(CommandPlayAbility::descriptor()->name());
```

Comment:
Why should we use descriptor()? I can find documentation for GetDescriptor, but not for descriptor.

Reply:
Old protobuf versions don't support static `GetDescriptor`.